JobBars/Cooldowns/Manager/CooldownManager.cs

```diff
@@ -26,10 +26,17 @@ public unsafe partial class CooldownManager : PerJobManager<CooldownConfig[]> {
             }
         }
 
-        public CooldownConfig[] GetCooldownConfigs( JobIds job ) {
+        public CooldownConfig[] GetCooldownConfigs( JobIds job, byte level ) {
             List<CooldownConfig> configs = new();
             if( JobToValue.TryGetValue( job, out var props ) ) configs.AddRange( props );
             if( CustomCooldowns.TryGetValue( job, out var customProps ) ) configs.AddRange( customProps );
+            if( level > 0 ) {
```

Comment:
there should also be a toggle for this, in case people don't want the cooldowns to get filtered

Reply:
Added a new option "Hide cooldowns of unavailable actions"